zerver/lib/digest.py

```diff
@@ -214,6 +225,13 @@ def get_hot_topics(
             hot_topics.append(topic)
         if len(hot_topics) == MAX_HOT_TOPICS_TO_BE_INCLUDED_IN_DIGEST:
             break
+    if use_new_ranking_system:
+        if message_weight and sender_weight is not None:
+            topics_by_score = sorted(topics, key=lambda dt: dt.score(message_weight, sender_weight))
+
+            return topics_by_score[:4:-1]
```

Comment:
I don't understand where the `4` and `-1` come from here; worth using a variable name or comment or something to clarify it.

Reply:
It is basically to get the <code>4 </code>topics with largest score.<code> -1 </code> is to count indexes in reverse order. I think to make it more readable I can use similar approach as e9587900e63fdc951c6f2b120cb8a3615b07abdc.